common/src/main/java/org/apache/celeborn/common/network/server/memory/MemoryManager.java

```diff
@@ -163,47 +161,33 @@ private MemoryManager(CelebornConf conf) {
                   "Memory manager actions transformed {} -> {}", lastAction, memoryManagerStat);
               if (memoryManagerStat == MemoryManagerStat.pausePushDataAndResumeReplicate) {
                 pausePushDataCounter.increment();
-                actionService.submit(
```

Comment:
You have changed the memory check interval. This needs more tests to ensure stability.